run-queue

```diff
@@ -85,6 +87,9 @@ def consume_task_queue(channel, amqp, declare_public_result, declare_rhel_result
         elif declare_rhel_result.method.message_count > 0:
             queue = ['public', 'rhel'][random.randrange(2)]
 
+    if statistics_queue and declare_stats_result.method.message_count > 0:
+        queue = [queue, 'statistics'][random.randrange(2)]
```

Comment:
This will prefer running things from the statistics queue (50% of the time, and public and rhel get 25% each). That's actually fine -- that queue is fast to process, and we *want* to prefer it. I just wonder if we shouldn't even bother with the 50% chance, and always process statistics first (same like the webhook queue, which is also drained first).

Reply:
Yeah, I realized that this would prefer stats queue. But I agree that it is fine and also kind-of wanted.
I would not go fully with draining stats queue first as it is **not that** important.